frontend/src/App.js

```diff
@@ -19,6 +19,8 @@ import { hasRole, useCurrentUser } from "main/utils/currentUser";
 import NotFoundPage from "main/pages/NotFoundPage";
 import AdminViewPlayPage from "main/pages/AdminViewPlayPage";
 import ProtectedPlayPage from "main/pages/ProtectedPlayPage";
+import AdminRemoveUserPage from "main/pages/AdminRemoveUserPage";
```

Comment:
```suggestion
import AdminSuspendUserPage from "main/pages/AdminSuspendUserPage";
```

Let's change this to "suspend user", per: #30